library-base/build.gradle

```diff
@@ -34,9 +34,9 @@ android {
 }
 
 dependencies {
-    provided 'io.reactivex.rxjava2:rxjava:2.1.4'
-    provided 'io.reactivex.rxjava2:rxandroid:2.0.1'
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compileOnly 'io.reactivex.rxjava2:rxjava:2.1.4'
```

Comment:
Updated to new dependency configurations as guided [here](https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#new_configurations)